sitescripts/notifications/parser.py

```diff
@@ -67,7 +70,9 @@ def _parse_notification(data, name):
     is_variant = current != notification
 
     if key == "inactive" and not is_variant:
-      current["inactive"] = True
+      if value.lower() not in BOOLEAN_STATES:
```

Comment:
I think it's safe to simply consider any value that isn't in `('', '0', 'false', 'no', 'off'')` truthful. That would simplify the logic a bit.